app/assets/javascripts/host_edit.js

```diff
@@ -142,10 +142,12 @@ function animate_progress(){
   if (stop_pooling == true) return;
   setTimeout(function() {
     var url = $('#host_progress_report_id').data('url');
-    $.get(url, function (response){
-       update_progress(response);
-       animate_progress();
-    })
+    if (url != null) {
```

Comment:
The best way is `if (typeof url != undefined){...}` (http://stackoverflow.com/a/519157/1091445).
You might also want to check if the double fire isn't caused by turbolinks - and then the correct fix would be to prevent turbolinks from handling the button click (see #2371 for example).